Sources/Corvus/Protocols/DTOs/DSL/EnumConnector.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import Fluent
+
+protocol AnyEnumConnector {
+    func set(on model: AnyModel)
+}
+
+@propertyWrapper
+public final class EnumConnector<Model, Value>: AnyEnumConnector
+    where Model: CorvusModel, Value: Codable & RawRepresentable, Value.RawValue == String
```

Comment:
Is adding those conformances to `Value` enough to support enums? Otherwise this looks the same as field, no?

Reply:
Basically yes. This is required by Model.Enum. My attempt to abstract all types (like FieldProperty, ParentProperty, ...) by using PropertyProtocol failed sadly.